app/views/application/index.html.erb

```diff
@@ -67,59 +66,140 @@
 	</div>
 </div>
 
-<div class="vertical-stripes">
-	<h1> News &amp; Highlights</h1>
+<div class="vertical-stripes" style="margin-top: 63px;text-align: center;width: 100%;">
+	<h1 style="white-space: nowrap;padding:14px;font-size: 250%;letter-spacing:.15rem;background-color:white!important;width: 25%;margin:auto;height:84px"> News &amp; Highlights</h1>
 </div>
 
+<% unless @highlights.empty? %>
```

Comment:
All this highlights code feels like it should be broken out into a partial. 

Also, it contains quite a bit of duplication. Do you think it could be refactored to loop over the existing highlights and reuse a single highlight partial?